### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by janjaap**
Adds a note on using Git/SourceTree for creating releases.

### CODE REVIEW

File: versioning/README.md [hunk 1]
```diff
@@ -52,3 +54,5 @@ Sources:
 + [mojombo/semver](https://github.com/mojombo/semver/blob/master/semver.md)
 + [olivierlacan/keep-a-changelog](https://github.com/olivierlacan/keep-a-changelog/blob/gh-pages/CHANGELOG.md)
  
+### Github/SourceTree
+Github supports creating releases, Git doesn't. That is why SourceTree isn't able to handle semantic versioning. For every release that needs to be created, don't do it in SourceTree.
```
Comment (alfakappa) [inline versioning/README.md]: Well... Remco presented this in his sheets as if it was possible. (Since you can add a release number to it actually)

If it is really not possible within souretree (which I really think would be really weird), please provide a tutorial then how to achieve it in git/command line.

File: versioning/README.md [hunk 2]
```diff
@@ -35,6 +35,8 @@ A release is initially pushed to an acceptance or staging server. The postfix '-
 
 Each change to the release candidate increments 'n'. So, the first RC for the major version 2.0.0 is 2.0.0-RC1. The second 2.0.0-RC2. The third 2.0.0-RC3 and so on. A push to production removes the '-RCn' postfix.
 
+A release indicates a halt on feature development.
+
```
Comment (alfakappa) [inline versioning/README.md]: is it? you can still develop features and decide later on if you want to have them in another release or not?

File: versioning/README.md (hunk 1 above)
Comment (janjaap) [inline versioning/README.md]: No, getting it done is your own responsibility. This is not the repo to provide a tut.

File: versioning/README.md (hunk 2 above)
Comment (janjaap) [inline versioning/README.md]: Of course you can, but that doesn't mean you should. If a release is issued, all effort should be made to finish that. So, ideally, feature development should not be addressed.

File: versioning/README.md (hunk 1 above)
Comment (janjaap) [inline versioning/README.md]: @alfakappa You can test it for yourself if you finish a hot-fix.

File: versioning/README.md (hunk 2 above)
Comment (janjaap) [inline versioning/README.md]: @timbeks What do you think?

Comment (timbeks) [inline versioning/README.md]: Does this mean that you set a specific number of features for a release and finish does first before you start on a new release?

Comment (janjaap) [inline versioning/README.md]: Yes

Comment (rmoolenaar) [inline versioning/README.md]: Hmm, if you do:
- Build features
- Decide what should be in the release
- Merge those features back to develop
- Build release

Then you can continue working on features.

However, then we have troubles with the peer review schema, because then we would like to merge immediately

Comment (timbeks) [inline versioning/README.md]: If it is on scrum base, you have a specific amount of features set already, so there shouldn't be more new features that are out of scope for the sprint. So I agree with @janjaap; You should not start a new feature that is out of scope for the sprint.

Comment (janjaap) [inline versioning/README.md]: Not really, because the flow should be:
- decide what needs to go in the next release,
- build feature(s)
- peer review -> merge feature branch(es) into develop
- create release branch off of develop,
- add improvements, no new features, to the release branch (release hardening),

After creating the release branch, new features can be merged into develop. Finishing a release will merge that branch into develop as well as master.

Comment (timbeks) [inline versioning/README.md]: That is fine, but the release branch should be created at the end of the sprint to finalize the feature branch. So depending on how large the release is, it should have a day or two to be able to finalize the branch and sprint. So than the team will only test and do some improvements on the release branch. There shouldn't be new features added at that time.

Comment (janjaap) [inline versioning/README.md]: +1

Comment (alfakappa) [inline versioning/README.md]: I think everybody is saying the same here :)

in the release branch no extra features can be added. Only hardening will be done.

extra features may be under development, but will never be added to the release branch. (they may however be added to develop branch)

I think that sums it up?

File: versioning/README.md (hunk 1 above)
Comment (alfakappa) [inline versioning/README.md]: I will do so and also will create a release branch for a project. (I will use Solvus for now, since we will release a first version tomorrow). See how it will work out in versioning. Come back to you later.

File: versioning/README.md (hunk 2 above)
Comment (janjaap) [inline versioning/README.md]: It does.

File: versioning/README.md (hunk 1 above)
Comment (rmoolenaar) [inline versioning/README.md]: To add a remark on your question @alfakappa : The release schema @janjaap is using adds extra possibilities OUTSIDE of Git. Stuff like release notes / changeling CANNOT be done via SourceTree.
SourceTree is a Git tool, not a Github tool.

File: versioning/README.md (hunk 2 above)
Comment (alfakappa) [inline versioning/README.md]: please add/edit in readme file, then I will merge

**Comment by janjaap** [comment]
Can we merge this in?

**Comment by alfakappa** [comment]
jep fine with me. For those who want. I made a release for solvus repo. please check if the labels are sufficient or not. it is done within source tree
